operators/pkg/instance-controller/common_logic.go

```diff
@@ -0,0 +1,70 @@
+package instance_controller
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+	v1 "k8s.io/api/core/v1"
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	crownlabsv1alpha2 "github.com/netgroup-polito/CrownLabs/operators/api/v1alpha2"
+	instance_creation "github.com/netgroup-polito/CrownLabs/operators/pkg/instance-creation"
+)
+
+// BuildInstanceObjectMeta creates ObjectMeta for Instance.
+func BuildInstanceObjectMeta(
```

Comment:
There is already a function configuring the labels [here](https://github.com/netgroup-polito/CrownLabs/blob/42968262487ce25c11be9d5c62651727f1bb0d17/operators/pkg/instance-creation/creation.go#L156)